### ISSUE TITLE
handle invalid <xi:include> href attributes

### ISSUE THREAD

**Issue by StevenClontz** [PreTeXtBook/pretext-cli#36]
As noted in https://github.com/rbeezer/mathbook/issues/1324 the `.xinclude()` method seems to silently pass through any `<xi:include/>` tags with invalid `href` attributes. Instead, I think that the build should fail when this happens with more useful feedback to the user.

**Comment by StevenClontz** [comment]
Researching here https://www.w3.org/TR/xinclude-11/#fallback_element it seems this is a bug(?) with lxml -- the resource error should have become a fatal error.

**Comment by rbeezer** [comment]
Didn't know about `xi:fallback`!  Always see it in these error messages, but thought it was some alternate content.  (I guess it is.)

This is **source** so not a place where we can scribble on the console.  Just drop a "Some content should be here!", but this is the author's responsibility then....

Not fatal at xsltproc command-line it seems....

Rob

**Comment by StevenClontz** [comment]
So for clarity, in https://github.com/rbeezer/mathbook/issues/1324 you noted this output on the console

```
/home/rob/mathbook/mathbook/doc/guide/basics/basics-part.xml:95: 
element include: XInclude error : could not load 
/home/rob/mathbook/mathbook/doc/guide/basics/foo.ptx, and no fallback was found
```

So you're saying that it doesn't break the build, but at least you get some feedback (compared to the 0 feedback we're getting with our CLI at the moment), yes?

**Comment by rbeezer** [comment]
On 6/26/20 9:44 AM, Steven Clontz wrote:
> So you're saying that it doesn't break the build, but at least you get some 
> feedback (compared to the 0 feedback we're getting with our CLI at the moment), yes?

Exactly.

**Comment by oscarlevin** [comment]
I am pretty sure this is fixed as part of the xml validation done for #66 .
